### PR TITLE
Resume and added art types

### PR DESCRIPTION
**Pull request by Lunatixz**
Please review art types... might have missed something? Thanks

### CODE REVIEW

File: resources/lib/gui.py [hunk 1]
```diff
@@ -1149,9 +1235,14 @@ def _close( self ):
...
 
     def onPlayBackEnded( self ):
         self.gui._trailerstopped()
 
     def onPlayBackStopped( self ):
         self.gui._trailerstopped()
+
+    def onPlayBackStarted( self ):   
+        self.seekTime( float( self.resume ) )
+        self.gui._close()
```
Comment (ronie) [inline resources/lib/gui.py]: is this needed?
no objection if it's a must-have, but please make an exclusion for trailer playback.
kodi should return to the globalsearch window after the playback of a movie trailer has finished.

**Comment by ronie** [comment]
looks good to me, apart from my question about closing the gui.

File: resources/lib/gui.py (hunk 1 above)
Comment (Lunatixz) [inline resources/lib/gui.py]: ahhh IC what you mean... I didn't catch that the dialog stays open for trailers... I'll move it back.

**Comment by ronie** [comment]
cool thanx!

**Pull request merged**